tests/test_cluster.py

```diff
@@ -2645,15 +2648,15 @@ def test_blocked_arguments(self, r):
             r.pipeline(transaction=True)
 
         assert (
-            str(ex.value).startswith("transaction is deprecated in cluster mode")
-            is True
+                str(ex.value).startswith("transaction is deprecated in cluster mode")
+                is True
         )
 
         with pytest.raises(RedisClusterException) as ex:
             r.pipeline(shard_hint=True)
 
         assert (
-            str(ex.value).startswith("shard_hint is deprecated in cluster mode") is True
+                str(ex.value).startswith("shard_hint is deprecated in cluster mode") is True
```

Comment:
Same here. Consider using a VS code python plugin that can take care of this. Like Sonarlint.